src/C++/CMakeLists.txt

```diff
@@ -71,12 +71,27 @@ else()
   target_link_libraries(${PROJECT_NAME} ${OPENSSL_LIBRARIES} ${MYSQL_CLIENT_LIBS} ${PostgreSQL_LIBRARIES} pthread)
 endif()
 
-target_include_directories(${PROJECT_NAME} PRIVATE ${CMAKE_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/src/C++)
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/ DESTINATION include/quickfix
```

Comment:
I'm getting a recursion error on this line for the file `OrderCancelReplaceRequest.h`, after including it in my project as mentioned in the PR description. Any reasons why?

```
CMake Error at vendor/quickfix/src/C++/CMakeLists.txt:74 (file):
  file COPY cannot copy file
  "/home/jsoares/Projects/atlas/fix_listener_cpp/vendor/quickfix/src/C++/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/fix43/OrderCancelReplaceRequest.h"
  to
  "/home/jsoares/Projects/atlas/fix_listener_cpp/vendor/quickfix/src/C++/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/include/quickfix/fix43/OrderCancelReplaceRequest.h".
```

Reply:
It could happen if you use 'in-source' build. I'll fix it.

Reply:
It works now. Thanks!